R/toolstartmessage.R

```diff
@@ -15,12 +15,18 @@
 #' @examples
 #' 
 #' \dontrun{
```

Comment:
I believe in they way you have written the example it should work out of the box and you can remove the `\dontrun` statement